server/src/internalClusterTest/java/org/elasticsearch/reservedstate/service/FileSettingsServiceIT.java

```diff
@@ -138,11 +141,30 @@ public static void writeJSONFile(String node, String json, AtomicLong versionCou
         Files.createDirectories(fileSettingsService.watchedFileDir());
         Path tempFilePath = createTempFile();
 
-        String settingsFileContent = Strings.format(json, version);
-        Files.write(tempFilePath, settingsFileContent.getBytes(StandardCharsets.UTF_8));
-        logger.info("--> Before writing new settings file with version [{}]", version);
-        Files.move(tempFilePath, fileSettingsService.watchedFile(), StandardCopyOption.ATOMIC_MOVE);
-        logger.info("--> After writing new settings file: [{}]", settingsFileContent);
+        logger.info("--> before writing JSON config to node {} with path {}", node, tempFilePath);
+        logger.info(Strings.format(json, version));
```

Comment:
Nit: maybe extract Strings.format(json, version) to a variable like in the original code?